### ISSUE TITLE
dense_Dense1 to have shape [,57,10], but got array with shape [1,1960,1].

### ISSUE THREAD

**Issue by plska** [Thoughtscript/x_team_tensorflow_js#1]
Hi I always get 
tf.js:5899 Uncaught (in promise) Error: Error when checking target: expected dense_Dense1 to have shape [,57,10], but got array with shape [1,1960,1].

**Comment by Lipe991** [comment]
+1

**Comment by Thoughtscript** [comment]
Hey! Yes, this is a WIP! It'll be the first TensorFlow.js stock prediction example!

**Comment by Thoughtscript** [comment]
Please also see: https://github.com/llSourcell/Financial_Forecasting_with_TensorflowJS/issues/1 which is "forked" from this repo!

**Comment by Thoughtscript** [comment]
Hi All! 

I'm @Thoughtscript - I wanted to apologize about the confusion this has caused! Yikes! 

First, let me be very clear that I'm not in any way affiliated with @llSourcell - see [this](https://github.com/llSourcell/Financial_Forecasting_with_TensorflowJS/).

This code was **NOT PRODUCTION WORTHY** and **INCOMPLETE** - as noted on the original repo listed [here](https://github.com/Thoughtscript/x_team_tensorflow_js) - it's a **WORK IN PROGRESS**.

Actually, I'm not sure how it ended up in a video or attracting any interest whatsoever. 

The code was being completed for an article to be posted on the top-ranked consulting firm [X-Team](https://x-team.com/blog/). Unfortunately, the company decided to move in another direction with it's writing and I've since moved over to Microsoft (though I remain affiliated with X-Team's great community).

Every other repo listed under my @Thoughtscript account without a *WIP* flag should be fully functional (with any issues, if they exist, noted). This particular project (which you'll note bears my name on in the pictures), is actually no longer maintained.

Any questions, feel free to send me an email at adam.gerard@gmail.com. Sorry for any confusion this has caused.

Cheers!

**Comment by Thoughtscript** [comment]
PS - maybe try:

1. Normalizing the values
1. Correcting the shapes
1. Correctly configuring the convolutional neural net
1. Leveraging parallel processing and WebGL since that shit's gonna break on single-threaded JS
